LayoutTests/http/tests/workers/service/shownotification-invalid-data.html

```diff
@@ -0,0 +1,58 @@
+<head>
+<script src="/resources/js-test-pre.js"></script>
```

Comment:
If you include `/resources/js-test-pre.js`, then you need to include `/resources/js-test-post.js` too at the end.